src/uint/extra_trait_impls/convert.rs

```diff
@@ -0,0 +1,48 @@
+macro_rules! convert_internal {
```

Comment:
I think this is added to address issue #220, but `resize()` should suffice for the conversion.